less/mixins.less

```diff
@@ -0,0 +1,13 @@
+.a-font {
+
+    text-decoration: none;
+    font-size: .95rem;
+
+}
+
+.p-font {
```

Comment:
Don't forget to put parentheses when creating a mixin because when your LESS file is compiled to CSS, it will include the mixins you created and they'll be treated as classes.